### PR TITLE
Update origin version to 1.2.0

### PR DESCRIPTION
**Pull request by praveenkumar**
Update openshift origin tag from 1.1.0 to 1.2.0 (Did a sanity test on older ADB box)

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@praveenkumar We should merge this PR as soon as ADB Vagrant box get released with adb-utils 1.7.0. As it has the fix for OpenShift registry and router setup i.e. https://github.com/projectatomic/adb-utils/pull/120

**Comment by praveenkumar** [comment]
@LalatenduMohanty if that looks good to you, it's ready to merge.

**Comment by LalatenduMohanty** [comment]
@praveenkumar will it work fine with ADB 2.1.0 box too without https://github.com/projectatomic/adb-utils/pull/120 ?

**Comment by praveenkumar** [comment]
> will it work fine with ADB 2.1.0 box too without projectatomic/adb-utils#120 ?

No it will not work so adb-utils-1.7 should be required.

File: components/centos/centos-openshift-setup/Vagrantfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ DOCKER_REGISTRY="docker.io"
 
 # The name of the OpenShift image.
```
Comment (LalatenduMohanty) [inline components/centos/centos-openshift-setup/Vagrantfile]: @praveenkumar can you please change the comment to make it explicit that we are talking about Docker hub tags here?

**Pull request merged**